### PR TITLE
Register noor.is-a.dev

### PR DESCRIPTION
**Pull request by noorudd-in**
Register noor.is-a.dev with MX record pointing to noor.is-a.dev.

### CODE REVIEW

File: domains/noor.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "noorudd-in",
+           "email": "",
+           "discord": "595988594464653338",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.mA_rtPTrYEwA7pi6nfzCRBgwDOyJiz8jKM-_Fo93G1Z6ekOQu4DTn2ITRu2_JrljDZEuWhNnlH8zktQGGJNezV42QYH5lOvdkSibOi6bK7gfqELntxBbBj50FHk4b2u7A3kPYPMlQsJCpEAne23roHjsfRr5bI51hkdpNUqcZmvUev2pFoqXvE_-gAwWPbamkdXZYc3hH2C7_YdyMdR4_zkJJMYoDDQzFy-eDtF0-mO7-_WzqvbXVl67MKxJz8p2DgfpdofuXD5GVPU5mCaiQZkxXbMzkNNMTnAjkpjQVmJ3T1ilhoIIzwU-SeV3R4vn40ZKKCsGCryutoueRkM9BQ.jQMhQu0O-pIkELaWx7gTmQ.BFVkgWEP3Vbmq5LPly97pdyKad4VxeS72HF037zcS4x1JN86AkM6Zo3vk48usJDF3Ye2xL1VbQBHgC6j23Jwjgh7kbztrf3bPT8D7i3mdpo.WhLwVU_TjVghzvskBQvPqA"
+        },
+    
+        "record": {
+            "MX": ["noor.is-a.dev"]
```
Comment (DEV-DIBSTER) [inline domains/noor.json:10]: 1) MX Records alone doesn't do anything.
2) Why is the record value the same as the domain you're requesting?

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/noor.json (hunk 1 above)
Comment (noorudd-in) [reply domains/noor.json:10]: I have shared hosting, let me know what should I add so I can add this domain to my hosting.

Comment (DEV-DIBSTER) [reply domains/noor.json:10]: Your record value needs to be changed, as it's exactly the same as the domain you're requesting. It needs to be an actual MX server.

Comment (noorudd-in) [reply domains/noor.json:10]: I couldn't find the MX records. But here is a A record.
Name:  noor.is-a.dev
TTL: 14400 
Type: A
record: 116.202.33.18

Hope this helps.

Comment (DEV-DIBSTER) [reply domains/noor.json:10]: Yeah, noor.is-a.dev is the same domain you're request. You're trying to create noor.json, which means the subdomain you're requesting is noor.is-a.dev. The domain is being pointed to the main site as it hasn't been claimed. The problem is your records. Your site won't point anywhere.

**Pull request closed without merging**